src/main/java/gregtech/loaders/recipe/chemistry/ReactorRecipes.java

```diff
@@ -696,5 +696,18 @@ public static void init() {
                 .fluidInputs(Chlorine.getFluid(1000))
                 .output(dust, Salt, 2)
                 .duration(200).EUt(VA[LV]).buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder()
+                .fluidInputs(Propene.getFluid(1000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .fluidInputs(CarbonMonoxide.getFluid(1000))
+                .fluidOutputs(Butraldehyde.getFluid(1000))
+                .duration(200).EUt(VA[HV]).buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder()
+                .fluidInputs(Butraldehyde.getFluid(1000))
+                .fluidInputs(PolyvinylAcetate.getFluid(1000))
+                .fluidOutputs(PolyvinylButyral.getFluid(1000))
+                .duration(400).EUt(VA[HV]).buildAndRegister();
```

Comment:
I think we should do this in ingot mols. Maybe:
250L Butyraldehyde + 144L PVA -> 144L PVB